x-pack/plugins/ml/public/application/trained_models/models_management/pipelines/expended_row.tsx

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
```

Comment:
This file should be `expanded_row.tsx` :)

Reply:
Fixed in d54dc9d170e211ba3fbfbc0aef969d12657b64fb